lib/generators/fae/templates/views/static_page_form.html.slim

```diff
@@ -1,6 +1,5 @@
 = simple_form_for @item, url: fae.update_content_block_path(slug: @item.slug), method: :put do |f|
-  header.content-header.js-content-header
-    == render 'fae/shared/form_header', header: @item
+  == render 'fae/shared/form_header', header: @item, hide_flash: true
```

Comment:
I know we'll likely need `hide_flash` (or `show_flash`) to support legacy  apps, but I'm wondering if we should have available on all forms by default. Can you think of why it would be omitted on the static page forms?

Reply:
Did some digging, and the errors partial was added to the `_form.html.slim` generator in 057485a5 . All the models that didn't have it (e.g. `acclaims` or `coaches`) were generated before 057485a5 . So I'll go back and set those to show flash messages in this PR.

However, for the static_pages, those have never had the flash messages partial. Those errors are handled inline, per form field. I guess it would only make it more cohesive if they all had flash messages.